src/hotspot/cpu/x86/stubGenerator_x86_64_tanh.cpp

```diff
@@ -324,13 +325,19 @@ address StubGenerator::generate_libmTanh() {
   __ enter(); // required for proper stackwalking of RuntimeStub frame
 
   __ bind(B1_2);
+  __ pextrw(rax, xmm0, 3);
+  __ shll(rax, 16);
+  __ pextrw(rax, xmm0, 4);
```

Comment:
There is an output dependency here, result of shll will be overwritten by following pextrw.

Reply:
Thanks - will address in the next update. Using multiple _pextrw_ commands is unnecessary anyway.